ecosystem/sep-0024.md

```diff
@@ -697,6 +695,18 @@ For example:
 }
 ```
 
+## Changes from SEP-6
+
+There is a small set of changes when upgrading from SEP-6 to SEP-24.
+
+1. SEP-24 now requires authentication on all endpoints.  The `authentication_required` flag is removed from the `/info` endpoint since authentication is assumed.
+1. `GET /deposit` and `GET /withdraw` have been replaced with [`POST /transactions/deposit/interactive`](#deposit) and [`POST /transactions/withdraw/interactive`](#withdraw).  This is for security purposes to keep submitted personally identifiable information out of URL query parameters.
+1. Remove the `type` parameter from the `/deposit` and `/withdraw` endpoints as this is information collected in the interactive flow.
+1. Removed `external_extra` transaction property since this should all live in a human readable ` more_info_url`.
+1. Changed the response of the deposit and withdraw endpoints from 403 to 200 since this is the expected flow.
+1. `/transactions` and `/transaction` are now required endpoints.
+1. `Transaction.more_info_url` is now a required property.
```

Comment:
I think some other fields also became required, like stellar_transaction_id and amount_in, amount_out, and others?